src/JsonUtil.h

```diff
@@ -195,57 +197,6 @@ namespace JsonUtil
 		}
 	}
 
-	template<class T>
-	static void DeserializeArrayValues(vector<T> &v, const Json::Value &root)
-	{
-		v.clear();
-		for( unsigned i=0; i<root.size(); i++ )
-		{
-			T t;
-			if( root[i].TryGet( t ) )
-				v.push_back( t );
-		}
-	}
-
-	// don't pull in the set header here
-	template<typename S, typename T>
-	static void DeserializeArrayValuesIntoSet(S &s, const Json::Value &root)
-	{
-		s.clear();
-		for( unsigned i=0; i<root.size(); i++ )
-		{
-			T t;
-			if( root[i].TryGet( t ) )
-				s.insert( t );
-		}
-	}
-
-	template<typename T>
-	static void DeserializeArrayValuesIntoVector(vector<T> &v, const Json::Value &root)
-	{
-		v.clear();
-		for( unsigned i=0; i<root.size(); i++ )
-		{
-			T t;
-			if( root[i].TryGet( t ) )
-				v.push_back( t );
-		}
-	}
```

Comment:
None of the removed functions were used except for this one, specialized with type RString.